### PR TITLE
Dg documentation

### PR DESCRIPTION
**Pull request by OhJunMing**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -210,18 +211,16 @@ unlikely to have) - `*`
...
 | `* * *`  | user                                       | filter by address   | find customers who live near a specific region
 | `* * *`  | user                                       | filter by name   | find customers whose name match most closely the phone number I remember
-| `* * *`  | user                                       | filter by Date Of Birth   | find customers whose DoB match most closely the phone number I remember
+| `* * *`  | user                                       | filter by Date Of Birth   | find customers whose Date of Birth match most closely the phone number I remember
 | `* * *`  | user                                       | filter by email   | find customers whose email match most closely the phone number I remember
-| `* * *`  | user                                       | filter by phone number   | find customers whose phone number match most closely the phone number I remember
+| `* * *`  | user                                       | filter by phone number   | find customers whose phone number match most closely the name I remember
```
Comment (nighoggDatatype) [inline docs/DeveloperGuide.md:216]: This is wrong, pls revert

Comment (rajobasu) [inline docs/DeveloperGuide.md:216]: name?? should be phone number

**Review by nighoggDatatype — COMMENTED** [review]
Aborted review

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -195,13 +195,10 @@ public CustomerListPanel getCustomerListPanel() {
      */
     private CommandResult executeCommand(String commandText) throws CommandException, ParseException {
         try {
```
Comment (rajobasu) [inline src/main/java/seedu/address/ui/MainWindow.java:197]: good catch on these debug statements

**Review by rajobasu — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (OhJunMing) [reply docs/DeveloperGuide.md:216]: if i change to phone number

find customers whose phone number match most closely the phone number I remember
dont make sense

**Review by rajobasu — APPROVED** [review]

**Pull request merged**